modules/collection/programs/zoxide.nix

```diff
@@ -42,26 +42,19 @@ in {
 
   config = mkIf cfg.enable {
     packages = mkIf (cfg.package != true) [cfg.package];
-    files = {
-      ".zshrc" = mkIf (config.rum.programs.zsh.enable && cfg.integrations.zsh.enable) {
-        text = mkAfter ''eval "$(${getExe cfg.package} init zsh ${toFlags})"'';
-      };
-    };
-    xdg.config.files = {
-      /*
-      Needs to be added to the end of the shell configuration files, hence the `mkIf` and `mkAfter`.
-      https://github.com/ajeetdsouza/zoxide#installation
-      */
```

Comment:
Do we want to retain this? I don't know how obvious/non-obvious the mkAfter stuff is.